src/sentry/utils/pytest/selenium.py

```diff
@@ -326,6 +326,16 @@ def snapshot(self, name, mobile_only=False):
                     self.driver.find_element_by_tag_name("body").screenshot(
                         u"{}/{}.png".format(snapshot_dir, slugify(name))
                     )
+                    has_tooltips = self.driver.execute_script(
+                        "window.__openAllTooltips && window.__openAllTooltips()"
```

Comment:
Does this need an explicit `return`?

Reply:
Oh right, keep forgetting, I'll give it a shot.